File: 179-pocket-change/solve.py
def count_change(change):
    initial_total = sum(change)

    base = initial_total // 100
    coins = initial_total % 100

    coins_formatted = f"{coins:02d}"

    return f"${base}.{coins_formatted}"

print(count_change([25, 10, 5, 1])) 

File: 179-pocket-change/solve.js
function countChange(change) {
  let total;
  let initialTotal = change.reduce((acc, current) => acc + current, 0);
  const base = Math.floor(initialTotal / 100);
  // let coinsFormatted = Number(initialTotal % 100).toFixed(2);
  let coins = initialTotal % 100;
  let coinsFormatted = coins.toString().padStart(2, '0');

  return `$${base}.${coinsFormatted}`;
}

console.log(countChange([25, 10, 5, 1]))